### PR TITLE
fix: improve pair detail on advanced trading view

### PR DESCRIPTION
**Pull request by berteotti**
# Summary

Improve pair detail UX on advanced trading view

image

### CODE REVIEW

**Comment by 0xVenky** [comment]
Looks great ! 🚀

**Review by 0xVenky — APPROVED** [review]
Looks great! :)

**Comment by Diogomartf** [comment]
the spacement seems little off.

image

image

**Comment by Diogomartf** [comment]
image

It would be great to have format easier to read, like $43,768.12. 

At least the dollar should move to the front.

File: src/pages/Swap/AdvancedSwapMode/PairDetails/PairDetails.component.tsx [hunk 1]
```diff
@@ -16,6 +16,17 @@ interface PairDetailsProps {
   activeCurrencyOption: Token | null | undefined
 }
 
+const RelativePrice = ({ relativePrice }: { relativePrice: string }) => {
```
Comment (Diogomartf) [inline src/pages/Swap/AdvancedSwapMode/PairDetails/PairDetails.component.tsx:19]:
```suggestion
const RelativePrice = ({ price }: { price: string }) => {
```

The relative price function receives a price,  and returns a relative price. So I think price, is a better name for the argument.

**Comment by MilanVojnovic95** [comment]
Tested and looks ok to me 👍

**Review by Diogomartf — APPROVED** [review]

**Pull request merged**